charts/dapr/charts/dapr_config/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Dapr configuration
 name: dapr_config
-version: 0.6.0
+version: 1.1.0-rc.1
```

Comment:
@yaron2 @artursouza Is this one safe to change? It is not referenced directly, but instead by some magic in `_helpers.tpl`.